deploy/compose/fetch-and-convert-ubuntu-img/fetch.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+# This script is designed to download a cloud image file (.img) and then convert it to a .raw.gz file.
+# This is purpose built so non-raw cloud image files can be used with the "image2disk" action.
```

Comment:
Some day if we can get qemuimg2disk action images shipped it would be nice to not need this step.